include/hpcombi/hpcombi.hpp

```diff
@@ -30,3 +34,73 @@
 #include "vect_generic.hpp"
 
 #endif  // HPCOMBI_HPCOMBI_HPP_
+
+/*! \mainpage HPCombi
```

Comment:
Seems to me that it might be better to just include `README.md` file as we were previously here, otherwise there's a good chance that the contents of this section and the `README.md` will become out of sync of over time. Was there a reason for having both? Might it not be better to just update `README.md` to include the extra material that's included here?

Reply:
The contents of this section and the README.md contain distinct information, without repetition. There is no need to check for synchronization. No risk of out of sync.

Indeed, that was precisely the point of this modification : do not copy paste, do not let the reader notice (after having wasted some time) that they have already read this file. Avoid circular link where https://libsemigroups.github.io/HPCombi/ advises to go read "The Doxygen auto generated [API](https://libsemigroups.github.io/HPCombi/)" and actually points to itself.

Alternatively one could move the contents of "/*! \mainpage HPCombi" (in hpcombi.hpp) into the readm, and in the main page of the Doxygen doc just *point* to the readme (instead of duplicating it). I prefered to separate the authors/contributors/thanks list and the technical information, but feel free to implement this alternative solution.